internal/server/server-definition.go

```diff
@@ -0,0 +1,465 @@
+// Package server provides primitives to interact with the openapi HTTP API.
+//
+// Code generated by github.com/deepmap/oapi-codegen version v1.12.2 DO NOT EDIT.
```

Comment:
I'd argue that it makes sense to keep this in version control. By doing that our CI definitions are simpler (no need to install tools to generate the server definition).

Reply:
I'd argue[^1] that ultimately we'd at least like our continuous integration workflow to warn us if there is a mismatch between the OpenAPI definitions and the generated code (e.g. if we forget to update them before committing), and that would also require us to install the same tools and perform the same operations.

[^1]: Just because that's something I enjoy, as you may have noticed. 😸